packages/valory/skills/market_manager_abci/graph_tooling/requests.py

```diff
@@ -168,16 +176,26 @@ def _handle_response(
         self._fetch_status = FetchStatus.SUCCESS
         return res
 
-    def _fetch_bets(self) -> Generator[None, None, Optional[list]]:
+    def _fetch_bets(self) -> Generator[None, None, Tuple[Optional[list], Callable]]:
         """Fetch questions from the current subgraph, for the current creators."""
         self._fetch_status = FetchStatus.IN_PROGRESS
 
-        query = questions.substitute(
-            creators=to_graphql_list(self._current_creators),
-            slot_count=self.params.slot_count,
-            opening_threshold=self.synced_time + self.params.opening_margin,
-            languages=to_graphql_list(self.params.languages),
-        )
+        if isinstance(self.current_subgraph, PolymarketGammaSubgraph):
+            # TODO: we ignore 'creators' and simply fetch 100 latest markets.
+            query = questions_polymarket_gamma.substitute(
```

Comment:
Pending TODO.

Reply:
Add 
`AND created_by=$creator `

to the "where:" query parameter on polymarket_gamma query

Reply:
It cannot be addressed this way since we do not know the creators in advance. Instead, fetching all the markets and adding pagination might make more sense.